docs/admin/authorization/rbac.md

```diff
@@ -78,42 +86,44 @@ metadata:
   name: read-pods
   namespace: default
 subjects:
-  - kind: User # May be "User", "Group" or "ServiceAccount"
-    name: jane
+- kind: User
+  name: jane
+  apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: Role
   name: pod-reader
   apiGroup: rbac.authorization.k8s.io
 ```
 
-`RoleBindings` may also refer to a `ClusterRole`. However, a `RoleBinding` that
-refers to a `ClusterRole` only applies in the `RoleBinding`'s namespace, not at
-the cluster level. This allows admins to define a set of common roles for the
-entire cluster, then reuse them in multiple namespaces.
+A `RoleBinding` may also reference a `ClusterRole` to grant the permissions to namespaced
+resources defined in the `ClusterRole` within the `RoleBinding`'s namespace.
+This allows administrators to define a set of common roles for the entire cluster,
+then reuse them within multiple namespaces.
 
 For instance, even though the following `RoleBinding` refers to a `ClusterRole`,
 "dave" (the subject) will only be able read secrets in the "development"
-namespace, the namespace of the `RoleBinding`.
+namespace (the namespace of the `RoleBinding`).
 
 ```yaml
 # This role binding allows "dave" to read secrets in the namespace "development"
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: read-secrets
-  namespace: development # This binding only applies in the "development" namespace
+  namespace: development # This only grants permissions within the "development" namespace
 subjects:
-  - kind: User # May be "User", "Group" or "ServiceAccount"
-    name: dave
+- kind: User
+  name: dave
+  apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: ClusterRole
   name: secret-reader
   apiGroup: rbac.authorization.k8s.io
 ```
 
-Finally a `ClusterRoleBinding` may be used to grant permissions in all
-namespaces. The following `ClusterRoleBinding` allows any user in the group
-"manager" to read secrets in any namespace.
+Finally, a `ClusterRoleBinding` may be used to grant permission at the cluster level and in all
```

Comment:
- [ ] Would this mean the same thing if you instead wrote the following?  
> at the cluster level, across all namespaces.

I'm not sure if you meant to draw the distinction between referring to resources that are not namespaced as well as namespaced resource, but cutting across all namespaces for the latter.